content/concepts/tokens/nftoken-auctions.md

```diff
@@ -0,0 +1,64 @@
+---
+html: nftoken-auctions.html
+parent: non-fungible-tokens.html
+blurb: You can assign another account to mint NFTs in your stead.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# Running an NFT Auction
+
+There are several ways to run an auction, each with advantages and disadvantages.
+
+## Run the Auction Off the XRPL, Complete the Purchase on XRPL
+
+This flow is the most straightforward. Note that the `NFTokenOffer` objects can always be canceled by their creator, so it's not possible to implement a binding offer.
+
+1. Store your bids in a private database.
+2. You take a cut of the winning bid.
+3. Send the buyer/seller the XRPL transaction to complete the purchase. 
+
+## Run the Auction in Brokered Mode, with a Reserve
+
+Run the auction in brokered mode, as an auction with a reserve.
+
+![Auction in Brokered Mode with a Reserve](img/nft-auction1.png "Auction in Brokered Mode with a Reserve")
+
+1. The Seller creates the NFT, then sets the auction reserve price using `NFTokenCreateOffer`, specifying the broker account as the destination.
+1. Buyers make bids using `NFTokenCreateOffer`, setting the broker account as the destination.
```

Comment:
nit: would this be considered Bidders instead of Buyers as they have not necessarily won the bid yet?

Reply:
on line 50 it does says `Bidders`, so i think it is meant to say it here as well

Reply:
Yes, updated.

Reply:
Changed "Buyers make bids" to "Bidders make offers," which is more accurate.